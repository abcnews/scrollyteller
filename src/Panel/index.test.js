const React = require('react');
const Renderer = require('react-test-renderer');
const Panel = require('.');

describe('Panel', () => {
  it('renders', () => {
    const tree = Renderer.create(<Panel />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders with some config', () => {
    const tree = Renderer.create(<Panel config={{ light: true, right: true }} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
