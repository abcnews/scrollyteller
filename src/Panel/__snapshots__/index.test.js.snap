// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel renders 1`] = `
<div
  className="base"
  id={undefined}
/>
`;

exports[`Panel renders with some config 1`] = `
<div
  className="base light right"
  id={undefined}
/>
`;
